### DISCUSSION TITLE
List of supported applications for media popup?

### DISCUSSION THREAD

**Discussion by headquarter8302** [ShankarBUS/ModernFlyouts#1141] [Q&A] [answered]
I see that VLC isn't supported, Chromium browsers (tested on Chrome & Edge) work just fine but lacks track controls, Windows 10's Groove Music (Media Player in W11) displays normally but lacks any type of control at all.

I wonder, what is Flyouts using so it can control media, and if it's hardcoded, a list of supported applications?

**Comment by prakharb5** [comment, marked as answer]
@headquarter8302 you can read more here
https://github.com/ModernFlyouts-Community/ModernFlyouts/blob/main/docs/GSMTC-Support-And-Popular-Apps.md
